llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -622,6 +622,9 @@ class TargetTransformInfo {
     /// Don't allow runtime unrolling if expanding the trip count takes more
     /// than SCEVExpansionBudget.
     unsigned SCEVExpansionBudget;
+    /// Allow runtime unrolling multi-exit loops. Should only be set if the
+    /// target determined that multi-exit unrolling is profitable for the loop.
+    bool UnrollMultiExit;
```

Comment:
```suggestion
    bool RuntimeUnrollMultiExit;
```
To clarify that this affects only runtime unrolling?

Reply:
Done thanks